examples/phys/FlatteLineShape.inl

```diff
@@ -381,9 +381,8 @@ int main(int argv, char** argc)
 	 //======================================================
 
     //f0
-
-    auto coef_ref0 = hydra::Parameter::Create().Name("f0_RC").Value(f0_RC).Error(0.0001);
-    auto coef_imf0 = hydra::Parameter::Create().Name("f0_IM").Value(f0_IMC).Error(0.0001);
+    auto coef_ref0 = hydra::Parameter::Create().Name("f0_RC").Value(f0_RC).Error(0.0001).Limits(-100,100);
```

Comment:
Define the limits in the parameters in terms of the nominal values in the parameters.